### DISCUSSION TITLE
What is the meaning of FullStack in RealWorld?

### DISCUSSION THREAD

**Discussion by hanmango-o** [realworld-apps/realworld#1267] [Q&A]
I want to develop a mobile app with `flutter`. And also I want to use `SpringBoot` as a BackEnd.

In this case, can I use `flutter` and `SpringBoot` as a Full Stack?

There is no mobile in Full Stack category.....🥲

here is the link.
- https://codebase.show/projects/realworld?category=fullstack

**Comment by geromegrignon** [comment]
Hi you can use it as a FullStack example. An isolated mobile app would go under the Frontend category.
